### PR TITLE
#504 send options along with harmonization flows

### PR DESCRIPTION
**Pull request by dmcassel**

### CODE REVIEW

File: quick-start/src/test/java/com/marklogic/quickstart/service/FlowManagerServiceTest.java [hunk 1]
```diff
@@ -155,4 +172,99 @@ public void runMlcp() throws IOException, InterruptedException, JSONException {
...
+                if (percentComplete == 100)
+                {
+                    synchronized (monitor) {
+                        monitor.notify();
+                    }
+                }
+            }
+        });
+
+        // Wait for the flow to finish
```
Comment (aebadirad) [inline quick-start/src/test/java/com/marklogic/quickstart/service/FlowManagerServiceTest.java]: I really like the sync block here for waiting for it to complete. +1

**Comment by grechaw** [comment]
Hi @dmcassel , Mike had told me that this PR implemented the passing of target database... I don't think it does though.  This PR pushes options down into the headers of the harmonized document, but the code that chooses target database doesn't look in the options to do anything.  Is there some oversight?  I'm trying to help Mike connect the dots from the UI to the services, and to understand the codebase better myself.

**Comment by dmcassel** [comment]
@grechaw it doesn't ... it allows users to pass along arbitrary options that will be used in a harmonization flow. The framework itself pays no attention to them. That was the goal of #504.

**Comment by grechaw** [comment]
Thanks Dave for that confirmation.  I think all the folks are on the same page about this request now

**Pull request merged**